build_config_linux.py

```diff
@@ -0,0 +1,51 @@
+#!python
+from os.path import expanduser
+from build_config import *
+
+home = expanduser("~")
+QT_DIR = home + '/Qt/5.7/gcc_64'
```

Comment:
Being able to parameterize the Qt version will save a lot of work later. Give that a thought.